apps/judicial-system/web/src/components/AccordionItems/IndictmentsCaseFilesAccordionItem/IndictmentsCaseFilesAccordionItem.tsx

```diff
@@ -163,11 +172,44 @@ const renderChapter = (chapter: number, name: string) => (
 )
 
 const CaseFile: React.FC<CaseFileProps> = (props) => {
-  const { caseFile, index, onReorder, onOpen } = props
+  const { caseFile, index, onReorder, onOpen, onRename } = props
+  const { formatMessage } = useIntl()
   const y = useMotionValue(0)
   const boxShadow = useRaisedShadow(y)
   const controls = useDragControls()
-  const [isDragging, setIsDragging] = useState(false)
+  const [isDragging, setIsDragging] = useState<boolean>(false)
+  const [editFileId, setEditFileId] = useState<string>()
+  const [editedFilenames, setEditedFilenames] = useState<EditedFileName[]>([])
```

Comment:
I don't quite get why we need to keep `editedFilenames` in a state if we are not rendering them anywhere. Can we discuss this?